### PR TITLE
removed course offering page on the homepage and added them to the course offering page

### PR DESCRIPTION
**Pull request by voidii**
Also added nav bar and tutor assignments to the course offering page

### CODE REVIEW

**Review by pconrad — DISMISSED** [review]
There are some places where merge conflicts were not resolved, and as a result, this code no longer compiles, and thus it fails on CI.

I suggest updating this branch on localhost, and doing a `mvn clean compile` and then a `mvn test` to get it into a state where it at least compiles before we look at it further.

**Comment by pconrad** [comment]
@voidii please see the comments above

**Comment by voidii** [comment]
OK, I used vs code to open the files and it seems like there are some changes that were not accepted. Now I have accepted those changes and they compile nicely now. I will a second check on the localhost to make sure everything is alright and push the changes to our branch.

**Comment by voidii** [comment]
The code compiles fine now and is up to date with master branch and I have also removed the course offering table from the main page but it seems to not be able to direct to the course offering page that we created. It was working on the previous version but not anymore. I will look further into that and will later push a working version.

**Review by yinonrousso — CHANGES_REQUESTED** [review]
Courses offering table and tutor assignment table were removed from the homepage but gave no alternative route to access their individual pages. I may hold off on pulling this until a navigation button brings me to a specific tutor assignment page and course offering page.

**Comment by voidii** [comment]
Fix the page and nav bar, should work fine now

**Review by yinonrousso — CHANGES_REQUESTED** [review]
Need to resolve merge conflicts

**Comment by jingjingyu123** [comment]
conflict should be resloved

File: src/main/resources/templates/fragments/basic_form.html [hunk 1]
```diff
@@ -184,6 +184,8 @@ <h2 class="my-5">Course Offerings</h2>
         </div>
       </div>
     </div>
+  -->
+
```
Comment (yinonrousso) [inline src/main/resources/templates/fragments/basic_form.html]: Please remove commented out code

File: src/main/resources/templates/fragments/basic_form.html [hunk 2]
```diff
@@ -239,6 +241,8 @@ <h2 class="my-5">Time Slots</h2>
         </div>
       </div>
     </div>
+
+    <!--
```
Comment (yinonrousso) [inline src/main/resources/templates/fragments/basic_form.html]: Same here rm commented out code

**Review by yinonrousso — APPROVED** [review]
I removed left over commented out code. The rest looks good. May want to add better navigation to new course offerings page

File: src/main/resources/templates/fragments/basic_form.html (hunk 1 above)
Comment (yinonrousso) [reply src/main/resources/templates/fragments/basic_form.html]: I removed the comments myself

**Pull request merged**